pages/index.css

```diff
@@ -21,3 +21,19 @@
 
 /* Visually Hidden Mix */
 @import url(../blocks/visually-hidden/visually-hidden.css);
+
+/*Block Footer*/
+@import url(../blocks/footer/footer.css);
+@import url(../blocks/footer/__author/footer__author.css);
+@import url(../blocks/footer/__author-logo/footer__author-logo.css);
+@import url(../blocks/footer/__container/footer__container.css);
+@import url(../blocks/footer/__copyright/footer__copyright.css);
+@import url(../blocks/footer/__link/footer__link.css);
+@import url(../blocks/footer/__social-media/footer__social-media.css);
+@import url(../blocks/footer/__social-icon/footer__social-icon.css);
+@import url(../blocks/footer/__author-link/footer__author-link.css);
+@import url(../blocks/footer/__author-logo/footer__author-logo.css);
```

Comment:
@ElenaPodv, kindly remove the import on line 35, as it duplicates the one already present on line 28.

Reply:
Done

Reply:
Done